### DISCUSSION TITLE
Client for flutter app

### DISCUSSION THREAD

**Discussion by subhendukundu** [edumeet/edumeet#840] [Ideas]
Hey, I was planning on putting together a [flutter](https://flutter.dev) client for this. I was go through the code, but realize I could save some time if I get some insight already.
So is there documentation to get started from the server perspective? or help me get started.

**Comment by misi** [comment]
Sounds great! Will it be open source?

You can also add it this repository. If you like send a PR. I would happy to merge here.
The app  core logic is in app/src/RoomClient.js. You can reuse it in your Flutter app.
For example see the @havfo SIP gw code that is also act like a UI-less client..: github.com/edumeet/edumeet-sip

**Comment by subhendukundu** [comment]
Yes it will be opensource, you guys did a great job on this, I would love to send a PR back. I see that could complete the project towards the next steps.
Perfect, I am looking at `app/src/RoomClient.js` now. Might put together a few questions I have any and send it over. That might be helpful for documentation as well.

**Comment by misi** [comment]
Just let us know if there is any question. We will try to help us much us possible..

**Comment by pranav-ted** [comment]
Hey guys,
Any progress on this?
